docs/swagger.yaml

```
@@ -666,6 +666,24 @@ paths:
       summary: Get all users for superadmin
       tags:
       - user
+  /user/first:
+    post:
+      produces:
+      - application/json
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 8 but found 6

(indentation)